doc/contributing/docs/markup/code.rst

```diff
@@ -1,8 +1,49 @@
 Writing about code
 ==================
 
+When writing articles, you have to format code specially, separating it from
+other text. This document will guide you through typical cases when
+it is recommended to use code highlighting.
+
+Defining what code is
+---------------------
+
+It is not difficult to recognize the code snippet in the text. First of all,
```

Comment:
Here "not difficult" is a subjective evaluation. It's a good practice to avoid evaluations but instead provide facts, so that our readers could make the same evaluation themselves.

Thing which isn't difficult for us, can be difficult for the reader. Saying that something is easy only adds to their frustration, when they struggle with doing something with the help of our docs. 

I highly recommend this talk by Jim Fisher: https://youtu.be/gsT2BBWBVmM